app/Services/OrganizationService.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+use App\DTO\OrganizationDTO;
+use App\Models\Organization;
+use App\Models\User;
+
+class OrganizationService
+{
+    public function create(OrganizationDTO $organization): void
+    {
+        Organization::create([
+            "name" => $organization->name,
+            "github_id" => $organization->github_id,
+            "avatar_url" => $organization->avatar_url,
+        ]);
+    }
+
+    public function removeMember(array $member, array $organizationData): void
+    {
+        $user = User::where("github_id", $member["id"])->first();
+        $organization = Organization::where("github_id", $organizationData["id"])->first();
```

Comment:
```suggestion
        $user = User::query()->where("github_id", $member["id"])->first();
        $organization = Organization::query()->where("github_id", $organizationData["id"])->first();
```

Reply:
Changed it. Why is this way better?

Reply:
Its for IDE, better recogniton of class methods.